exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/xsort/ExternalSortBatch.java

```diff
@@ -110,11 +111,12 @@
   private LinkedList<BatchGroup> batchGroups = Lists.newLinkedList();
   private LinkedList<BatchGroup> spilledBatchGroups = Lists.newLinkedList();
   private SelectionVector4 sv4;
-  private FileSystem fs;
+  private static FileSystem fs;
```

Comment:
looks like the github markdown killed part of you message, it did get through to the email they sent out, copied modified versions of the paths here:

   root spill dir / query_id / major_frag_id / minor_frag_id / operator_id/

Instead we should make it:

    root spill dir / query_id__major_frag_id__minor_frag_id__operator_id /

I don't think we need the double underscores in the path, I just did that to separate the components of the directory name.